infra/stg/docker-compose.yml

```diff
@@ -72,7 +72,7 @@ services:
       - public-web
 
   sapungobrol-victoriametrics:
-    image: victoriametrics/victoria-metrics:latest
+    image: victoriametrics/victoria-metrics:v1.106.0

```

Comment:
any reason to pin the version? or it's just to make sure that it won't break in the future?

Reply:
Yep, you got it. Having the `latest` tag was never the intent, I left that out by accident.